pkg/apiserver/apiserver.go

```diff
@@ -295,14 +295,14 @@ func installHandlers(c *ExtraConfig, s *genericapiserver.GenericAPIServer) {
 		m := controllernetworkpolicy.NewNetworkPolicyMutator(c.networkPolicyController)
 		// Install handlers for NetworkPolicy related mutation
 		s.Handler.NonGoRestfulMux.HandleFunc("/mutate/acnp", webhook.HandleMutationNetworkPolicy(m))
-		s.Handler.NonGoRestfulMux.HandleFunc("/mutate/anp", webhook.HandleMutationNetworkPolicy(m))
+		s.Handler.NonGoRestfulMux.HandleFunc("/mutate/annp", webhook.HandleMutationNetworkPolicy(m))
```

Comment:
It's perhaps better to support the old path for some releases, just in case the CRD manifest and the image are not upgraded together (not sure if there is such case, but at least harmless to do it).

Reply:
Kept the old path in handler and `allowedPath`.